reactor-core/src/main/java/reactor/core/publisher/FluxElapsed.java

```diff
@@ -46,7 +46,7 @@ public void subscribe(CoreSubscriber<? super Tuple2<Long, T>> actual) {
 	@Override
 	public Object scanUnsafe(Attr key) {
 		if (key == Attr.RUN_ON) return scheduler;
-
+		if (key == Attr.THREAD_MODIFIER) return true;
```

Comment:
`ElapsedSubscriber` is synchronous and does not change the thread. The scheduler is used, but only for timestamping

Reply:
I haven't go through the Flux classes at the moment, only those which are linked to Mono du to conversion needs. So this will be removed anyway, but thanks for noticing the wrong value 😉